lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2305,7 +2305,118 @@ definitions:
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      ho_algorithm_config:
```

Comment:
This should be explicitly marked as optional. Please add some details on specifications that other developers or API users could look up to understand how to use these fields correctly.

Reply:
![1640509029661_773D068C-61E9-461a-A918-38F399500364](https://user-images.githubusercontent.com/87301024/147403951-ecf53daf-e2f4-4d16-b883-47fc97ab50d9.png)
Modified, thank you!